lib/touch.js

```diff
@@ -129,58 +129,57 @@ const Injector = ObjC.registerClass({
 
         const app = UIApplication.sharedApplication();
         const event = app['- _touchesEvent'].call(app);
-        event['- _clearTouches'].call(event);
-
-        const absoluteTime = mach_absolute_time();
-
-        const timestamp = [
-            absoluteTime.shr(32).toNumber(),
-            absoluteTime.and(0xffffffff).toNumber()
-        ];
-
-        touch.setTimestamp_(CFGetSystemUptime());
-
-        const eventMask = (phase === UITouchPhaseMoved)
-            ? kIOHIDDigitizerEventPosition
-            : (kIOHIDDigitizerEventRange | kIOHIDDigitizerEventTouch);
-        const isRangeAndTouch = (phase !== UITouchPhaseEnded) ? 1 : 0;
-
-        const hidEvent = IOHIDEventCreateDigitizerFingerEvent(
-            kCFAllocatorDefault,
-            timestamp,
-            0,
-            2,
-            eventMask,
-            point[0],
-            point[1],
-            0,
-            0,
-            0,
-            isRangeAndTouch,
-            isRangeAndTouch,
-            0);
-
-        if ('- _setHidEvent:' in touch) {
-          touch['- _setHidEvent:'].call(touch, hidEvent);
-        }
-
-        event['- _setHIDEvent:'].call(event, hidEvent);
-        event['- _addTouch:forDelayedDelivery:'].call(event, touch, false);
-
-        const pool = NSAutoreleasePool.alloc().init();
-        try {
-          app.sendEvent_(event);
-        } finally {
-          pool.release();
+        if (event) {
```

Comment:
```suggestion
        if (event !== null) {
```

(For consistency with the existing style.)